content/en/tracing/trace_collection/custom_instrumentation/android/otel.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: Android and Android TV Custom Instrumentation using OpenTelemetry API
 description: 'Instrument your Android and Android TV application with OpenTelemetry API, to send traces to Datadog.'
+aliases:
+- /tracing/trace_collection/otel_instrumentation/android/
+- /tracing/trace_collection/custom_instrumentation/otel_instrumentation/android
```

Comment:
Were these docs pages previously and were removed? We usually only add aliases when we remove a page and want to redirect the link to another page.

Reply:
These never existed, but all the tiles in `layouts/partials/apm/apm-otel-instrumentation-custom.html` use the same `/otel_instrumentation/xxx` alias so I just created one for Android too for the sake of consistency.

Reply:
Thanks, do you mind removing the aliases since we don't need them and in case in the future if we do want to add pages with those links we won't have a problem?

Reply:
Updated, however I'm not 100% sure of which path to use to reference the right page from the tiles layout, so feel free to change this directly if I did a mistake 🙏